src/js/common/index.js

```diff
@@ -5,9 +5,6 @@
 // other such libraries in here. Most of the site does not use these legacy
 // frameworks and it belongs in a lower-level module.
 
-// Global site style.
-require('../../sass/style.scss');
```

Comment:
@crwallace and @bshyong can you both also take a look at this change and confirm that moving the import of `style.scss` from `index.js` to `no-react-entry.js` works?